rskj-core/src/main/java/co/rsk/net/light/LightProcessor.java

```diff
@@ -0,0 +1,222 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2020 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package co.rsk.net.light;
+
+import co.rsk.core.RskAddress;
+import co.rsk.crypto.Keccak256;
+import co.rsk.db.RepositorySnapshot;
+import co.rsk.db.RepositoryLocator;
+import co.rsk.net.light.message.*;
+import org.ethereum.core.*;
+import org.ethereum.net.message.Message;
+import org.bouncycastle.util.encoders.Hex;
+import org.ethereum.db.BlockStore;
+import org.ethereum.db.TransactionInfo;
+import org.ethereum.vm.DataWord;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nonnull;
+import java.util.LinkedList;
+import java.util.List;
+
+
+/**
+ * Created by Julian Len and Sebastian Sicardi on 21/10/19.
+ */
+public class LightProcessor {
+    private static final Logger logger = LoggerFactory.getLogger("lightprocessor");
+    // keep tabs on which nodes know which blocks.
```

Comment:
This comment seems off. That's not what the BlockStore does, or is it?

Reply:
You're right, this seems to be an old comment, we should remove it, it makes no sense there